packages/mobile/src/screens/edit-track-screen/screens/ReleaseDateScreen.tsx

```diff
@@ -122,11 +134,22 @@ export const ScheduledReleaseRadioField = (props) => {
           .month(releaseDateMoment.month())
           .day(releaseDateMoment.day())
       }
-
+      if (newReleaseDate.isAfter(moment())) {
```

Comment:
nit: maybe simplify and refactor this since it's the same logic in the `handleDateChange` function. no biggie tho